scripts/line_length.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+from typing import Optional
+from typing import Sequence
+from subprocess import call
+
+# We recommend a maximum line length of 80, but do allow up to 100 characters
+# if deemed necessary by the developer. Lines that exceed that limit will
+# be wrapped after 80 characters automatically.
+lineLengthTheshold = 100
```

Comment:
pep says: "Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL."

kLineLengthThreshold also seems fine. 

https://www.python.org/dev/peps/pep-0008/#constants